lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -169,6 +169,20 @@ public function getDropDatabaseSQL($name)
     }
 
     /**
+     * Returns the SQL statement for closing currently active connections on the given database.
+     *
+     * This is useful to force DROP DATABASE operations which could fail because of active connections.
+     *
+     * @param string $database The name of the database to close currently active connections for.
+     *
+     * @return string
+     */
+    public function getCloseActiveDatabaseConnectionsSQL($database)
```

Comment:
Possibly a better solution if the schema manager is platform specific (and won't rely on the platform abstraction anyway).